src/System.CodeDom/tests/System/CodeDom/Compiler/CompilerResultsTests.cs

```diff
@@ -30,20 +30,61 @@ public void Ctor_TempFileCollection(TempFileCollection tempFiles)
         }
 
         [Fact]
-        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Assembly.Load ignores CodeBase by design, except on netfx. See https://github.com/dotnet/coreclr/issues/10561")]
-        public void CompiledAssembly_ValidPathToAssembly_ReturnsExpected()
+        public void CompiledAssembly_GetWithPathToAssembly_ReturnsExpected()
         {
-            var results = new CompilerResults(null) { PathToAssembly = typeof(int).Assembly.EscapedCodeBase };
-            Assert.Equal(typeof(int).Assembly, results.CompiledAssembly);
+            var results = new CompilerResults(null) { PathToAssembly = typeof(CompilerResultsTests).Assembly.Location };
+
+            Assert.NotNull(results.CompiledAssembly);
+//            Assert.Equal(typeof(CompilerResultsTests).Assembly, results.CompiledAssembly);
```

Comment:
Loaded assemblies are same, somehow this test failing, maybe just compare assembly names?

>  Assert.Equal() Failure
>       Expected: System.CodeDom.Tests, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
>       Actual:   System.CodeDom.Tests, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51

Reply:
It is one assembly loaded multiple times.

Yes, comparing assembly name would be fine

Reply:
Thank you!